ui.apps/src/main/content/jcr_root/apps/acs-commons/config.publish/org.apache.sling.jcr.repoinit.RepositoryInitializer-distribution.config

```diff
@@ -0,0 +1,11 @@
+scripts=[
+"# extend rights for installing the ui.content package containing /var nodes with AEMaaCS (https://helpx.adobe.com/in/experience-manager/kb/cm/cloudmanager-deploy-fails-due-to-sling-distribution-aem.html)
+create path /var/acs-commons(nt:folder)
+
+# AEM classic does not know this system user, but creating it below system/acs-commons shouldn't do any harm
+create service user sling-distribution-importer with path system/acs-commons
```

Comment:
@kwin Does this service user already exist on AEM CS? If it does, i assume this "creation" will noop right? (obviously, if it does exist OOTB - it wont exist at "/home/users/system/acs-commons")

Reply:
Yes, noop when user exists: https://github.com/apache/sling-org-apache-sling-jcr-repoinit/blob/5a14f4c3c96381f44310316689fdd4bd048bf436/src/main/java/org/apache/sling/jcr/repoinit/impl/UserVisitor.java#L62